src/dataStrategies.js

```diff
@@ -121,7 +122,14 @@ export const terms_stats = ({
       )
     )
     done = true
-    return result.context.terms
+
+    let terms = _.get('context.terms', result)
+    if (include) {
+      let omittedFields = _.difference(['count', 'doc_count', 'min', 'max', 'avg', 'sum'], include)
```

Comment:
I don't think `sum` and `count` should be be an explicit opt-in. Those should be there by default since they're used by all the `terms_stats` callees.

Reply:
I think the best approach here would be to add `include` support to `terms_stats` in elasticsearch, because in there we can actually change the ES query so we don't even calculate the fields we don't need.

Reply:
@daedalus28 sounds like a good plan. I'll start looking at that and close this out for now as it's not clear to me how this will look when that's in.